npm-shrinkwrap.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "balena-etcher",
-  "version": "1.5.0",
+  "version": "1.5.2",
```

Comment:
1.5.2 and not 1.5.1 ?

Reply:
This file is generated, the version is copied from the `package.json` file which has `1.5.2` on master.

Reply:
Ahhh, I see from https://github.com/balena-io/etcher/commits/master that **resin-ci** is now bumping the version every time a PR gets merged.